components/SubMenu/index.vue

```diff
@@ -43,9 +43,7 @@ export default {
     }
   },
   mounted () {
-    if (this.$store.state.isSameRoutePage) {
-      this.activeMenu = this.$store.state.subMenu
-    } else {
+    if (Array.isArray(this.menus)) {
```

Comment:
hello @yoslie this code maybe work for normal case but if you try open detail in submenu testimonial and back, i think the exptectation is stay in submenu testimonial but actual state is got submenu hero banner so, the submenu can't really work for this code. so what do you think ?

Reply:
on my way to check it first, thankyou @adzharamrullah

Reply:
hi I already fixed it [here](https://github.com/jabardigitalservice/dedi-cms/pull/36/commits/253f5ce3beda3340100e5cf4351e7886bbebf9bf), please check it out @adzharamrullah thanks 🙏